e2e/stubs/fake-github.ts

```diff
@@ -202,7 +202,7 @@ export class FakeGitHub implements StubbedServer {
     const pattern = /\/users\/([^/]+)\/repos/;
     await this.server.forGet(pattern).thenCallback((request) => {
       const match = request.path.match(pattern);
-      const owner = match![1];
+      const owner = decodeURIComponent(match![1]);
```

Comment:
The square brackets in `github-actions[bot]` need to be decoded when it's sent in the url path.